.github/workflows/deploy.yml

```diff
@@ -64,6 +67,7 @@ jobs:
           EDL_PASSWORD: ${{ secrets.EDL_PASSWORD }}
           CLOUDFORMATION_ROLE_ARN: ${{ secrets.CLOUDFORMATION_ROLE_ARN }}
           MONTHLY_JOB_QUOTA_PER_USER: ${{ matrix.quota }}
+          AUTORIFT_NAMING_SCHEME: ${{ matrix.autorift_naming_scheme }}
```

Comment:
You'll have to add an input parameter to the deploy action to accept this variable:
https://github.com/ASFHyP3/hyp3/blob/naming-scheme-as-parameter/.github/actions/deploy-hyp3/action.yml#L4

Reply:
Nevermind I'm a dummy and missed it.